edx_exams/apps/api/serializers.py

```diff
@@ -40,6 +47,73 @@ class Meta:
         fields = ['name', 'verbose_name', 'lti_configuration_id']
 
 
+class CourseExamConfigurationWriteSerializer(serializers.ModelSerializer):
+    """
+    Serializer for writing to the CourseExamConfiguration model.
+
+    We have separate read and write serializers because the read serializer uses the name field of the provider instance
+    to populate the provider serializer field by using the source attribute. The write serializer, on the other hand,
+    accepts the name as the value for the provider field. If we use the same serializer, deserialized data gets
+    represented as {'provider': {'name': <provider>}}, which makes reading the data less clear.
+    """
+    provider = serializers.CharField(allow_null=True)
+    # The escalation_email is a nullable field, but we require that clients send us an escalation_email.
+    escalation_email = serializers.EmailField(allow_null=True, allow_blank=False, required=True)
```

Comment:
This is a change.

The original was `escalation_email = EmailField(allow_blank=True, max_length=254, required=False)`, which is the default generated by DRF.

The new one is `escalation_email = serializers.EmailField(allow_null=True, allow_blank=False, required=True)`.

The two differences are...
1. We now allow nulls and forbid blanks. Although this is not Django/DRF best-practice for character based fields, this more closely matches both the existing API on this endpoint, because `provider` is a nullable character field, and the Studio proctoring settings API, which accepts a `null` for the `escalation_email`. I have set `allow_blank=False` so that we only have one representation of the empty value.
2. This field is now required. I think it's a clearer API contract with the client for this value to always be required. Otherwise, the client is left wondering why a value is required sometimes and not other times. Sending `escalation_email: null` from the client is more explicit.